cmd/livepeer_cli/wizard_token.go

```diff
@@ -19,6 +21,27 @@ func (w *wizard) transferTokens() {
 		"amount": {fmt.Sprintf("%v", amount.String())},
 	}
 
+	var answer string
+	userAccepted := false
+	for !userAccepted {
+		fmt.Printf("Are you sure you want to send %s LPT to \"%s\"? [y/n]: ", val["amount"], val["to"])
```

Comment:
I see the number of LPTs is displayed inside brackets `[ ]`.
```
Are you sure you want to send [10] LPT to "[0x123]"?
```

Could you remove these brackets? Like this:
```
Are you sure you want to send 10 LPT to "0x123"?
```